README.md

```diff
@@ -1,70 +1,183 @@
+- **React setuo**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+  - React setup
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- **Starting project development from a template repo**
 
-## Learn More
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## 🚀 Live Demo <a name="live-demo"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- Not deployed yet
 
-### Code Splitting
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Analyzing the Bundle Size
+To get a local copy up and running, follow these steps.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Prerequisites
 
-### Making a Progressive Web App
+In order to run this project you need Webpack deveopment environment installed for this on you local machine
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Setup
 
-### Advanced Configuration
+Clone this repository to your desired folder on your local machine
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Install
+
+The projects incorporate some linters check, to get them running on your local system follow these below;
+
+1. `.github/workflows/linters.yml` is already included in project, this is used by Github to run tests on the repo.
+2. To install Webpack packages and dependencies
+
+- `npm init -y` initalize the project.
+- `npm i` installs all dependencies listed in the projects's package.json
+
+- `npm install webpack webpack-cli --save-dev` installs webpack
+- `npm install --save-dev html-webpack-plugin` installs webpack plugin
+- `npm install --save-dev style-loader css-loader` installs assets loaders
+- `npm install --save-dev webpack-dev-server` installs webpack server
```

Comment:
- [ ] There is no need for the user to install Webpack, kindly consider removing this section from the README so it doesn't create confusion